### PR TITLE
backlight module: allow chaging brightness with mouse scroll

### PR DESCRIPTION
**Pull request by ultrabug**
and setting initial brightness on start !

I was frustrated that the backlight module didn't provide the functionality so I figured that it was about time I got to code again :D

### CODE REVIEW

**Comment by ultrabug** [comment]
lol failed CI for being a bad kitty... Am I getting old ? (jk) ahah

File: .cache/v/cache/lastfailed [hunk 1]
```diff
@@ -0,0 +1 @@
+{}
```
Comment (tobes) [inline .cache/v/cache/lastfailed]: I'm not sure this should be here

File: py3status/modules/backlight.py [hunk 2]
```diff
@@ -3,19 +3,34 @@
 Display the current screen backlight level.
 
 Configuration parameters:
+    brightness_delta: Change the brightness by this step
+        (default 5)
+    brightness_initial: Set brightness to this value on start
+        (default 100)
```
Comment (tobes) [inline py3status/modules/backlight.py]: how about having this default as None, but if set then will set that value at startup?  That way we have one less parameter.

File: py3status/modules/backlight.py [hunk 3]
```diff
@@ -3,19 +3,34 @@
...
+    force_on_start: Should we set initial brightness on start
+        (default False)
     format: Display brightness, see placeholders below
         (default '☼: {level}%')
 
 Format status string parameters:
     {level} brightness
 
+Requires:
+    xbacklight: only for click events, not for brightness detection
```
Comment (tobes) [inline py3status/modules/backlight.py]: > click events

it is really for changing the brightness as this includes the initial setting

File: py3status/modules/backlight.py [hunk 4]
```diff
@@ -37,24 +52,55 @@ class Py3status:
...
         for brightness_line in open("%s/brightness" % self.device_path, 'rb'):
             brightness = int(brightness_line)
         for brightness_max_line in open("%s/max_brightness" % self.device_path, 'rb'):
             brightness_max = int(brightness_max_line)
+        return brightness * 100 // brightness_max
 
-        full_text = self.format.format(level=(brightness * 100 // brightness_max))
+    def backlight(self,  i3s_output_list, i3s_config):
+        level = self._get_backlight_level()
+        full_text = self.format.format(level=level)
```
Comment (tobes) [inline py3status/modules/backlight.py]: We should change this to use safe_format so we can have crazy formats

File: py3status/modules/backlight.py [hunk 5]
```diff
@@ -37,24 +52,55 @@ class Py3status:
...
+
+    def _get_backlight_level(self):
         for brightness_line in open("%s/brightness" % self.device_path, 'rb'):
             brightness = int(brightness_line)
         for brightness_max_line in open("%s/max_brightness" % self.device_path, 'rb'):
             brightness_max = int(brightness_max_line)
+        return brightness * 100 // brightness_max
 
-        full_text = self.format.format(level=(brightness * 100 // brightness_max))
+    def backlight(self,  i3s_output_list, i3s_config):
```
Comment (tobes) [inline py3status/modules/backlight.py]: maybe remove `,  i3s_output_list, i3s_config` as you're here

File: py3status/modules/backlight.py (hunk 4 above)
Comment (ultrabug) [inline py3status/modules/backlight.py]: done

File: py3status/modules/backlight.py (hunk 5 above)
Comment (ultrabug) [inline py3status/modules/backlight.py]: done

File: py3status/modules/backlight.py (hunk 3 above)
Comment (ultrabug) [inline py3status/modules/backlight.py]: indeed

**Comment by ultrabug** [comment]
Thanks, addressed your cool review !

**Pull request merged**